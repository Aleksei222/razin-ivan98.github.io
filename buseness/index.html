<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Business</title>
    <link rel="stylesheet" href="style.css">
    <script src="main.js"></script>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
</head>

<body>
    <header>
        <div class="container">
            <div class="flex-wrapper-header">
                <div class="logo"><img src="assets/Logo.svg" alt="logo"></div>
                <div class="navigation">
                    <nav>
                        <a class="menu-item" href="">home</a>
                        <a class="menu-item" href="">service</a>
                        <a class="menu-item" href="">about</a>
                        <a class="menu-item" href="">pricing table</a>
                        <a class="menu-item" href="">how it work</a>
                        <a class="menu-item" href="">happy clients</a>
                        <a class="menu-item" href="">contact us</a>
                    </nav>
                </div>
            </div>
        </div>
    </header>
    <section class="home">
        <div class="container">
            <h1 class="title home__title">Boost up your Local business</h1>
            <h5 class="subtitle home__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting
                industry. Lorem Ipsum typesetting</h5>
            <div class="home__form">
                <input placeholder="&#128100; Full Name" type="text">
                <input placeholder="&#9993; Email Address" type="text">
                <input placeholder="&#9743; Phone Number" type="text">
                <button class="home__btn">Request a quote</button>
            </div>
            <div class="home__laptop">
                <div class="laptop-image-container"><img src="assets/Vector_Smart_Object@2x.svg" alt="laptop">
                    <div class="slider">
                    </div>
                </div>
                <div class="dots-container">
                    <span class="dot dot-active" onclick="change()"></span>
                    <span class="dot"></span>
                    <span class="dot"></span>
                    <span class="dot"></span>
                </div>
            </div>
        </div>
    </section>
</body>

</html>