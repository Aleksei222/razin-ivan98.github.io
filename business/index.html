<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Business</title>
    <link rel="stylesheet" href="style.css">
    <script src="main.js"></script>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
</head>

<body>
    <header>
        <div class="container">
            <div class="flex-wrapper-header">
                <div class="logo"><img src="assets/Logo.svg" alt="logo"></div>
                <div class="navigation">
                    <nav>
                        <a class="menu-item" href="">home</a>
                        <a class="menu-item" href="">service</a>
                        <a class="menu-item" href="">about</a>
                        <a class="menu-item" href="">pricing table</a>
                        <a class="menu-item" href="">how it work</a>
                        <a class="menu-item" href="">happy clients</a>
                        <a class="menu-item" href="">contact us</a>
                    </nav>
                </div>
            </div>
        </div>
    </header>
    <section class="home">
        <div class="container">
            <h1 class="title home__title">Boost up your Local business</h1>
            <h5 class="subtitle home__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting
                industry. Lorem Ipsum typesetting</h5>
            <div class="home__form">
                <input placeholder="&#128100; Full Name" type="text">
                <input placeholder="&#9993; Email Address" type="text">
                <input placeholder="&#9743; Phone Number" type="text">
                <button class="home__btn">Request a quote</button>
            </div>
            <div class="home__laptop">
                <div class="laptop-image-container"><img id="laptop-image" src="assets/Vector_Smart_Object@2x.svg"
                        alt="laptop">
                    <div class="slider">
                    </div>
                </div>
                <div class="dots-container">
                    <span class="dot dot-active" onclick="change()"></span>
                    <span class="dot"></span>
                    <span class="dot"></span>
                    <span class="dot"></span>
                </div>
            </div>
        </div>
    </section>
    <section class="services">
        <div class="container">
            <h1 class="title services__title">our services</h1>
            <h5 class="subtitle services__subtitle">Lorem Ipsum is simply dummy Business industry.</h5>
            <div class="list services__list">
                <div class="list__item">
                    <img src="assets/services/1.png" alt="1">
                    <div>
                        <h4 class="list__title">Font-Awesome Icons</h4>
                        <p class="list__text">Lorem Ipsum is simply dummy text of the printing and typesetting ind
                            ustry. Lorem Ipsum has been the industry's standard dummy text ever
                            since the 1500s, when an unknown printer.</p>
                    </div>
                </div>
                <div class="list__item">
                    <img src="assets/services/2.png" alt="2">
                    <div>
                        <h4 class="list__title">Fully Responsive Design</h4>
                        <p class="list__text">Many desktop publishing packages and web page editors now use Lorem Ipsum
                            as their default model text, and a search for 'lorem ipsum' will uncover many web sites
                            still in their infancy</p>
                    </div>
                </div>
                <div class="list__item">
                    <img src="assets/services/3.png" alt="3">
                    <div>
                        <h4 class="list__title">Working Contact form</h4>
                        <p class="list__text">Many desktop publishing packages and web page editors now use Lorem Ipsum
                            as their default model text, and a search for 'lorem ipsum' will un
                            cover many web sites still in their infancy</p>
                    </div>
                </div>
                <div class="list__item">
                    <img src="assets/services/4.png" alt="4">
                    <div>
                        <h4 class="list__title">Retina Ready icons here</h4>
                        <p class="list__text">There are many variations of passages of Lorem Ipsum available, but the
                            majority have suffered alteration in some form, by injected humour
                            or randomised words which don't look even slightly believable.</p>
                    </div>
                </div>
                <div class="list__item">
                    <img src="assets/services/5.png" alt="5">
                    <div>
                        <h4 class="list__title">Friendly Code</h4>
                        <p class="list__text">Finibus Bonorum et Malorum" by Cicero are also reproduced in their
                            exact original form, accompanied by English versions from the 1914 translation by H. Rackham
                        </p>
                    </div>
                </div>
                <div class="list__item">
                    <img src="assets/services/6.png" alt="6">
                    <div>
                        <h4 class="list__title">Flat trendy landing page Design</h4>
                        <p class="list__text">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
                            Lorem Ipsum has been the industry's standard dummy text ever since
                            the 1500s, when an unknown printer.</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="about">
        <div class="left-background"></div>
        <div class="right-background">
            <div class="about__container">
                <h1 class="title about__title">about us</h1>
                <h5 class="subtitle about__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting
                    industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an
                    unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived
                    not only five centuries, but also the leap into electronic typesetting, remaining essentially
                    unchanged. It was popularised in the ore recently with desktop publishing software like Aldus
                    PageMaker including versions of Lorem Ipsum</h5>
                <button class="about__btn">Read more</button>
            </div>
        </div>
    </section>
    <section class="price">
        <div class="container">
            <h1 class="title services__title">our services</h1>
            <h5 class="subtitle services__subtitle">Lorem Ipsum is simply dummy Business industry.</h5>
            <div class="list price__list">
                <div class="list__item">
                    <h3 class="list__title">basic plan</h3>
                    <h1 class="cash">$22</h1>
                    <h5 class="list__subtitle">per mounth</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">1 Account</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">8 hour supports</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">Custom Cloud Service</h5>
                    <button class="price__btn">get started</button>
                </div>
                <div class="list__item">
                    <h3 class="list__title">premium plan</h3>
                    <h1 class="cash">$42</h1>
                    <h5 class="list__subtitle">per mounth</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">2 Account</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">100MB Storage</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">Custom Cloud Service</h5>
                    <button class="price__btn">get started</button>
                </div>
                <div class="list__item">
                    <h3 class="list__title">gold plan</h3>
                    <h1 class="cash">$62</h1>
                    <h5 class="list__subtitle">per mounth</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">8 Account</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">24 hour supports</h5>
                    <h5 class="list__subtitle list__subtitle_demarked">Custom Cloud Service</h5>
                    <button class="price__btn">get started</button>
                </div>
            </div>
        </div>
    </section>
</body>

</html>