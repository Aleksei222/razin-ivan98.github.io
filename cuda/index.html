<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="styles.css">
    <link href="https://fonts.googleapis.com/css?family=Titillium+Web&display=swap" rel="stylesheet">
    <title>Cuda</title>
</head>

<body>
    <header>
        <div class="container header__container">
            <div class="logo-container"><img src="assets/logo.svg" alt="logo"></div>
            <div class="nav-container">
                <nav class="menu">
                    <a href="#" class="menu__item">Home</a>
                    <a href="#" class="menu__item">About</a>
                    <a href="#" class="menu__item">Work</a>
                    <a href="#" class="menu__item">Blog</a>
                    <a href="#" class="menu__item">Contact</a>
                </nav>
            </div>
        </div>
    </header>
    <section class="home">
        <div class="container">
            <h1 class="title home__title">Hi there! We are the new kids on the block
                and we build awesome websites and mobile apps.</h1>
            <button class="btn home__btn">Work with us</button>
        </div>
    </section>
    <section class="services">
        <div class="container">
            <h1 class="title services__title">SERVICES WE PROVIDE</h1>
            <h6 class="subtitle services__subtitle">We are working with both individuals and businesses from all over
                the globe
                to create awesome websites and applications.
            </h6>
            <div class="list">
                <div class="list__item"><img src="assets/Flag_icon.svg" alt="flag">
                    <h4 class="list__title ">Branding</h4>
                    <p class="list__text">Lorem ipsum dolor sit amet,
                        consectetuer adipiscing elit, sed diam nonummy nibh.</p class="list__text">
                </div>
                <div class="list__item"><img src="assets/Crayon_icon.svg" alt="crayon">
                    <h4 class="list__title">Design</h4>
                    <p class="list__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem </p
                        class="list__text">
                </div>
                <div class="list__item"><img src="assets/Gears_icon.svg" alt="gears">
                    <h4 class="list__title">Development</h4>
                    <p class="list__text">At vero eos et accusamus et iusto odio dignissimos qui blanditiis praesentium.
                    </p class="list__text">
                </div>
                <div class="list__item"><img src="assets/Rocket_icon.svg" alt="rocket">
                    <h4 class="list__title">ROCKET SCIENCE</h4>
                    <p class="list__text">Et harum quidem rerum est et expedita distinctio. Nam libero tempore.</p>
                </div>
            </div>
        </div>
    </section>
    <section class="team">
        <div class="container">
            <h1 class="title team__title">MEET OUR BEAUTIFUL TEAM</h1>
            <h6 class="subtitle team__subtitle">We are a small team of designers and developers, who help brands with
                big ideas.
            </h6>
            <div class="list">
                <div class="list__item">
                    <div>
                        <img src="assets/Base.png" alt="photo">
                        <h4 class="list__title ">ANNE HATHAWAY</h4>
                        <p class="list__subsubtitle">CEO / Marketing Guru</p>
                        <p class="list__text">Lorem ipsum dolor sit amet,
                            consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore
                            magna.
                        </p class="list__text">
                    </div>
                    <div class="list icons-list">
                        <div class="list__item"><img src="assets/icons/Fb_icon.svg" alt="facebook"></div>
                        <div class="list__item"><img src="assets/icons/LinkedIn_icon.svg" alt="linkedIn"></div>
                        <div class="list__item"><img src="assets/icons/Mail_icon.svg" alt="mail"></div>
                        <div class="list__item"><img src="assets/icons/Twitter_icon.svg" alt="twitter"></div>
                    </div>
                </div>
                <div class="list__item">
                    <div>
                        <img src="assets/Base.png" alt="photo">
                        <h4 class="list__title ">Kate Upton</h4>
                        <p class="list__subsubtitle">Creative Director</p>
                        <p class="list__text">Duis aute irure dolor in in voluptate velit esse cillum dolore fugiat
                            nulla
                            pariatur. Excepteur sint occaecat non diam proident.</p class="list__text">
                    </div>
                    <div class="list icons-list">
                        <div class="list__item"><img src="assets/icons/Fb_icon.svg" alt="facebook"></div>
                        <div class="list__item"><img src="assets/icons/LinkedIn_icon.svg" alt="linkedIn"></div>
                        <div class="list__item"><img src="assets/icons/Mail_icon.svg" alt="mail"></div>
                        <div class="list__item"><img src="assets/icons/Twitter_icon.svg" alt="twitter"></div>
                    </div>
                </div>
                <div class="list__item">
                    <div>
                        <img src="assets/Base.png" alt="photo">
                        <h4 class="list__title ">Olivia Wilde</h4>
                        <p class="list__subsubtitle">Lead Designer</p>
                        <p class="list__text">Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia
                            consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p class="list__text">
                    </div>
                    <div class="list icons-list">
                        <div class="list__item"><img src="assets/icons/Fb_icon.svg" alt="facebook"></div>
                        <div class="list__item"><img src="assets/icons/LinkedIn_icon.svg" alt="linkedIn"></div>
                        <div class="list__item"><img src="assets/icons/Mail_icon.svg" alt="mail"></div>
                        <div class="list__item"><img src="assets/icons/Twitter_icon.svg" alt="twitter"></div>
                    </div>
                </div>
                <div class="list__item">
                    <div>
                        <img src="assets/Base.png" alt="photo">
                        <h4 class="list__title ">Ashley Greene</h4>
                        <p class="list__subsubtitle">SEO / Developer</p>
                        <p class="list__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
                            doloremque laudantium, totam rem aperiam.</p class="list__text">
                    </div>
                    <div class="list icons-list">
                        <div class="list__item"><img src="assets/icons/Fb_icon.svg" alt="facebook"></div>
                        <div class="list__item"><img src="assets/icons/LinkedIn_icon.svg" alt="linkedIn"></div>
                        <div class="list__item"><img src="assets/icons/Mail_icon.svg" alt="mail"></div>
                        <div class="list__item"><img src="assets/icons/Twitter_icon.svg" alt="twitter"></div>
                    </div>
                </div>

            </div>
        </div>
    </section>
    <section class="skills">
        <div class="container">
            <h1 class="title skills__title">WE GOT SKILLS!</h1>
            <h6 class="subtitle skills__subtitle">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
                eiusmod
                tempor incididunt ut labore et dolore magna aliqua.
            </h6>
            <div class="list">
                <div class="list__item"><img src="assets/skills/1.svg" alt="1">
                    <h4 class="list__title ">Web Design</h4>
                </div>
                <div class="list__item"><img src="assets/skills/2.svg" alt="2">
                    <h4 class="list__title ">HTML / CSS</h4>
                </div>
                <div class="list__item"><img src="assets/skills/3.svg" alt="3">
                    <h4 class="list__title ">GRAPHIC DESIGN</h4>
                </div>
                <div class="list__item"><img src="assets/skills/4.svg" alt="4">
                    <h4 class="list__title ">UI / UX</h4>
                </div>
            </div>
        </div>
    </section>
    <section class="portfolio">
        <div class="container">
            <h1 class="title portfolio__title">OUR PORTFOLIO</h1>
            <h6 class="subtitle portfolio__subtitle">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet
                consectetur, adipisci velit, sed quia non numquam
            </h6>
            <nav class="menu portfolio__menu">
                <a class="menu__item">ALL</a>
                <a class="menu__item">web</a>
                <a class="menu__item">apps</a>
                <a class="menu__item">icons</a>
            </nav>
            <div class="list portfolio__list">
                <div class="list__item"><img src="assets/portfolio/1.png" alt="1">
                    <h4 class="list__title ">Isometric Perspective Mock-Up</h4>
                </div>
                <div class="list__item"><img src="assets/portfolio/2.png" alt="2">
                    <h4 class="list__title ">Time Zone App UI</h4>
                </div>
                <div class="list__item"><img src="assets/portfolio/3.png" alt="3">
                    <h4 class="list__title ">Viro Media Players UI</h4>
                </div>
                <div class="list__item"><img src="assets/portfolio/4.png" alt="4">
                    <h4 class="list__title ">Blog / Magazine Flat UI Kit</h4>
                </div>
            </div>
            <button class="btn portfolio__btn">LOAD MORE PROJECTS</button>
        </div>
    </section>
    <section class="about">
        <div class="container">
            <h1 class="title about__title">WHAT POEPLE SAY ABOUT US</h1>
            <h6 class="subtitle about__subtitle">Our clients love us!
            </h6>
            <div class="list about__list">
                <div class="list__item"><img src="assets/Base.png" alt="photo">
                    <div>
                        <p class="list__cursive">“Nullam dapibus blandit orci, viverra gravida dui lobortis eget.
                            Maecenas
                            fringilla urna eu nisl scelerisque.”</p>
                        <h1 class="list__title">Chanel Iman</h1>
                        <p class="list__subsubtitle">CEO of Pinterest</p>
                    </div>
                </div>
                <div class="list__item"><img src="assets/Base.png" alt="photo">
                    <div>
                        <p class="list__cursive">“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In
                            condimentum facilisis porta.”</p>
                        <h1 class="list__title">ADRIANA LIMA</h1>
                        <p class="list__subsubtitle">Founder of Instagram</p>
                    </div>
                </div>
                <div class="list__item"><img src="assets/Base.png" alt="photo">
                    <div>
                        <p class="list__cursive">“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In
                            condimentum facilisis porta.”</p>
                        <h1 class="list__title">ANNE HATHAWAY</h1>
                        <p class="list__subsubtitle">Lead Designer at Behance</p>
                    </div>
                </div>
                <div class="list__item"><img src="assets/Base.png" alt="photo">
                    <div>
                        <p class="list__cursive">“Phasellus non purus vel arcu tempor commodo. Fusce semper, purus vel
                            luctus molestie, risus sem cursus neque.”</p>
                        <h1 class="list__title">EMMA STONE</h1>
                        <p class="list__subsubtitle">Co-Founder of Shazam</p>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="form">
        <div class="container">
            <h1 class="title form__title">GET IN TOUCH</h1>
            <h6 class="subtitle form__subtitle">1600 Pennsylvania Ave NW, Washington, DC 20500, United States of
                America. Tel: (202) 456-1111
            </h6>
            <div class="form__form">
                <div>
                    <input type="text" placeholder="Your Name*" id="name">
                    <input type="text" placeholder="Your Email*" id="email">
                </div>
                <textarea placeholder="Your Message*" name="comment" cols="40" rows="3"></textarea>
            </div>
            <button class="btn form__btn">SEND MESSAGE</button>
        </div>
    </section>
    <footer>
        <div class="container">
            <nav class="footer__menu">
                <a class="menu__item" href="">Facebook</a>
                <a class="menu__item" href="">Twitter</a>
                <a class="menu__item" href="">Google+</a>
                <a class="menu__item" href="">LinkedIn</a>
                <a class="menu__item" href="">Behance</a>
                <a class="menu__item" href="">Dribble</a>
                <a class="menu__item" href="">GitHub</a>
            </nav>
        </div>
    </footer>
</body>

</html>