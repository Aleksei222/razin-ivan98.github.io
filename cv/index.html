<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="style.css">
    <link href="https://fonts.googleapis.com/css?family=PT+Sans&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="font-awesome-4.7.0/font-awesome-4.7.0/css/font-awesome.min.css">
    <title>Ivan Razin CV</title>
</head>

<body>
    <div class="container">
        <section class="block about-me">
            <div class="about-me__header">
                <img src="logo.jpg" id="photo">
                <div class="about-me__titles">
                    <h1>Разин Иван Вадимович</h1>
                    <h3>Frontend разработчик</h3>
                </div>
            </div>
            <div class="about-me__body">
                <h2><i class="fa fa-user-circle" aria-hidden="true"></i> О себе</h2>
                <p class="text">
                    &emsp;Возраст: 21 год</p>
                <p class="text">
                    &emsp;Легко обучаюсь, хорошо адаптируюсь к новым условиям. Люблю познавать новое и работать в
                    коллективе.
                    В
                    любой работе проявляю творческий подход и всегда довожу дело до конца
                </p>
            </div>

        </section>
        <section class="block contacts">
            <h2><i class="fa fa-address-card" aria-hidden="true"></i>
                Контакты</h2>
            <p class="text"><i class="fa fa-phone" aria-hidden="true"></i>&ensp;
                +7(917)513-72-79</p>
            <p class="text"><i class="fa fa-envelope" aria-hidden="true"></i>&ensp;
                razin-ivan98@ya.ru</p>
            <p class="text"><i class="fa fa-github" aria-hidden="true"></i>&ensp;
                <a href="https://github.com/razin-ivan98/" target="_blank">razin-ivan98</a></p>
            <p class="text"><i class="fa fa-vk" aria-hidden="true"></i>&ensp;
                <a href="https://vk.com/vondarm/" target="_blank">vondarm</a></p>
        </section>
        <section class="block lang-skills">
            <h2><i class="fa fa-language" aria-hidden="true"></i>
                Знание естественных языков</h2>
            <p class="text"> &emsp;Носитель русского языка</p>
            <p class="text"> &emsp;Владею английским на уровне свободного чтения документации</p>

        </section>
        <section class="block education">
            <h2><i class="fa fa-graduation-cap" aria-hidden="true"></i> Образование</h2>
            <div class="list">
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/маи.png" alt="mai" class="list__logo">
                    </div>
                    <p class="text list__text">Учусь на четвертом курсе МАИ по
                        специальности "Проекирование, производство и эксплуатация ракет и РКК"</p>
                </div>
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/school21.png" alt="school21" class="list__logo">
                    </div>
                    <p class="text list__text">Прохожу обучение в школе
                        программирования School21</p>
                </div>
            </div>
        </section>
        <section class="block skills">
            <h2><i class="fa fa-pie-chart" aria-hidden="true"></i>
                Профессиональные навыки</h2>
            <div class="list">
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/html5.png" alt="html" class="list__logo"></div>
                    <p class="text list__text">Знаю основные теги и атрибуты, семантические элементы HTML5. Могу
                        сверстать форму
                        или
                        интегрировать встраиваемый контент</p>
                </div>
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/css3.png" alt="css" class="list__logo"></div>
                    <p class="text list__text">Умею работать с селекторами, знаю некоторые псевдоклассы. Понимаю, как
                        работают каскадирование, наследование и специфичность. Владею Flexbox и Grid Layout. Ознакомился
                        с методолгией БЭМ и уже пробую применять ее. Могу сверстать лэндинг из PSD макета </p>
                </div>
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/js.png" alt="js" class="list__logo"></div>
                    <p class="text list__text">
                        Владею основными конструкциями языка JavaScript. Могу отправить ajax запрос на сервер и внести
                        изменения в DOM. Также знаю основные алгоритмы и структуры данных. Могу определить сложность
                        алгоритма. Умею создавать Promise и обработчики событий
                    </p>
                </div>
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/webgl.png" alt="webgl" class="list__logo"></div>
                    <p class="text list__text">
                        Могу создать небольшую интерактивную иконку или целую 3D игру прямо в браузере с
                        распараллеливанием на видеокарту
                    </p>
                </div>
                <div class="list__item">
                    <div class="list__logo-container"><img src="logos/vue.png" alt="vue" class="list__logo"></div>
                    <p class="text list__text">
                        Могу создать SPA с помощью фреймворка VueJS. Понимаю принцип реактивности, умею связывать
                        элементы DOM и переменные. Умею использовать вычисляемые свойства и слежение, а также хуки
                        жизненного цикла Vue
                    </p>
                </div>
                <!-- <div class="list__item">
                    <img src="" alt="" class="list__logo">
                </div> -->
            </div>
        </section>


        <section class="block achivements">
            <h2><i class="fa fa-star" aria-hidden="true"></i> Мои достижения</h2>
            <p class="text achivements__text">
                <i class="fa fa-trophy" aria-hidden="true"></i> &emsp;Создал <a
                    href="https://mypicshare.000webhostapp.com" target="_blank">небольшую социальную сеть</a>. Фронт -
                на нативном JS. Бэк - на PHP, также без
                фреймворков, MVC. Пользователь
                может делать публикации с наложением стикеров на фото. Есть лайки, комментарии, а также диалоги. Большая
                часть действий производится без перезагрузки страницы. Есть мобильная версия
            </p>
            <p class="text achivements__text">
                <i class="fa fa-trophy" aria-hidden="true"></i> &emsp;Сверстал отзывчивый <a
                    href="https://razin-ivan98.github.io/golden/" target="_blank">сайт</a> по PSD
                макету. Не
                использовал flex и grid. Только float. При маленьком разрешении экрана навбар прячется в "гамбургер"
            </p>
            <p class="text achivements__text"><i class="fa fa-trophy" aria-hidden="true"></i> &emsp;
                Сделал еще один <a href="https://razin-ivan98.github.io/cuda/" target="_blank">сайт</a> по PSD макету.
                Для
                выравнивания и позиционирования элементов использовал Flexbox. Верстка отзывчива
            </p>

            <p class="text achivements__text"><i class="fa fa-trophy" aria-hidden="true"></i> &emsp;
                Сделал отзывчивый <a href="https://razin-ivan98.github.io/business/" target="_blank">сайт</a> с
                интерактивным слайдером.
            </p>
            <p class="text achivements__text"><i class="fa fa-trophy" aria-hidden="true"></i> &emsp;
                Написал простой <a href="https://razin-ivan98.github.io/newfdf/" target="_blank">редактор</a> 3D
                поверности при помощи
                Canvas API
            </p>
            <p class="text achivements__text"><i class="fa fa-trophy" aria-hidden="true"></i> &emsp;
                Создал на WebGL две интерактивные программы, графически отображающие фрактальные множества: <a
                    href="https://razin-ivan98.github.io/julia/" target="_blank">Julia</a> и
                <a href="https://razin-ivan98.github.io/raymarching/" target="_blank">Mandelbulb</a>. Во втором случае
                был реализован алгоритм RayMarching
            </p>
            <p class="text achivements__text"><i class="fa fa-trophy" aria-hidden="true"></i> &emsp;
                Разработал интерактивную <a href="https://razin-ivan98.github.io/dog/" target="_blank">программу</a>
                для просмотра 3D моделей в браузере с возмжностью наложения текстуры.
            </p>

            <p class="text">
                Все проекты адаптированы под мобильные устройства и опубликованы на Github Pages. Ознакомиться с ними
                можно по ссылкам выше
            </p>


        </section>

        <div class="block likes"></div>
    </div>

    </div>
</body>

</html>