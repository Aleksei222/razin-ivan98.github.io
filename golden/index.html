<!DOCTYPE html>

<html>

<head>
    <title>Golden</title>
    <link rel="stylesheet" href="styles.css">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>

<body>
    <div class="page">
        <section id="home">
            <header>
                <div class="logo-container">
                    <div class="logo"><img src="assets/logo.png"></div>
                </div>
                <div class="nav-container">
                    <div class="hamburger">
                        <input id="menu-toggle" type="checkbox" />
                        <label class="menu-btn" for="menu-toggle">
                            <span></span>
                        </label>
                        <nav>
                            <a class="nav-item" href="#home">Home</a>
                            <a class="nav-item" href="#services">Services</a class="nav-item">
                            <a class="nav-item" href="#portfolio">Portfolio</a class="nav-item">
                            <a class="nav-item" href="#about">About</a class="nav-item">
                            <a class="nav-item last-nav-item" href="#">Contacts</a class="nav-item">
                        </nav>
                    </div>


                </div>
            </header>
            <div class="welcome">
                <h3 class="welcome-h3">Welcome to our studio!</h3>
                <h1 class="welcome-h1">it's nice to meet you</h1>
                <button class="welcome-button">tell me more</button>
            </div>
        </section>
        <section id="services">
            <h3 class="services-h3">Services</h3>
            <h6 class="services-h6">Proin iaculis purus consequat sem cure. </h6>
            <div class="services-list">
                <div class="services-list-item"><img src="assets/icon1.png" alt="e-commerce">
                    <h5>E-Commerce</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
                <div class="services-list-item"><img src="assets/icon2.png" alt="responsive web">
                    <h5>Responsive Web</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
                <div class="services-list-item"><img src="assets/icon3.png" alt="web-security">
                    <h5>Web Security</h5>
                    <p>Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt.</p>
                </div>
            </div>
        </section>

        <section id="portfolio">
            <h3 class="services-h3">Our portfolio</h3>
            <h6 class="services-h6">Proin iaculis purus consequat sem cure. </h6>
            <div class="services-list portfolio-list">
                <div class="services-list-item"><img src="assets/img1.jpg" alt="img1">

                </div>
                <div class="services-list-item"><img src="assets/img2.jpg" alt="img2">

                </div>
                <div class="services-list-item"><img src="assets/img3.jpg" alt="img3">

                </div>
            </div>
        </section>

        <section id="about">
            <h3 class="services-h3">About us</h3>
            <h6 class="services-h6">Proin iaculis purus consequat sem cure. </h6>
            <div class="about-list">
                <div class="about-list-item">
                    <div class="about-list-item-left">
                        <h4 class="date">July 2010</h4>
                        <h4>Our Humble Beginnings</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt. Proin
                            iaculis purus consequat sem cure digni
                            ssim. Donec porttitora entum suscipit.</p>
                    </div>
                    <div class="about-list-item-center"><img src="assets/about1.jpg" alt="about1"></div>
                    <div class="about-list-item-right"></div>
                </div>
                <div class="about-list-item">
                    <div class="about-list-item-left"></div>
                    <div class="about-list-item-center"><img src="assets/about2.jpg" alt="about2"></div>
                    <div class="about-list-item-right">
                        <h4 class="date">January 2011</h4>
                        <h4>Facing Startup Battles</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt. Proin
                            iaculis purus consequat sem cure digni
                            ssim. Donec porttitora entum suscipit aenean
                            rhoncus posuere odio in tincidunt.</p>
                    </div>
                </div>
                <div class="about-list-item">
                    <div class="about-list-item-left">
                        <h4 class="date">December 2012</h4>
                        <h4>Enter The Dark Days</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt. Proin
                            iaculis purus consequat sem cure digni.</p>
                    </div>
                    <div class="about-list-item-center"><img src="assets/about3.jpg" alt="about3"></div>
                    <div class="about-list-item-right"></div>
                </div>
                <div class="about-list-item">
                    <div class="about-list-item-left"></div>
                    <div class="about-list-item-center"><img src="assets/about4.jpg" alt="about4"></div>
                    <div class="about-list-item-right">
                        <h4 class="date">february 2014</h4>
                        <h4>Our Triumph</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt. Proin
                            iaculis purus consequat sem cure digni
                            ssim. Donec porttitora entum suscipit aenean.</p>
                    </div>
                </div>
            </div>
        </section>
        <footer>
            <h5>Â© Copyright 2014 FreebiesXpress.com</h5>
        </footer>
    </div>
</body>

</html>